rt-scripts-optimizer.php

```diff
@@ -85,6 +85,19 @@ function rt_scripts_handler( $tag, $handle, $src ) {
 		return null;
 	}
 
+	/**
+	 * Checks if the plugin has to be disabled.
+	 *
+	 * Return true if it has to be disabled.
+	 *
+	 * @return bool.
+	 */
+	$disable_rt_optimzer = apply_filters( 'rt_disable_rt_optimzer', false );
```

Comment:
@can you change it to just `disable_rt_scripts_optimizer`

Reply:
resolved